### PR TITLE
Doc update for single-node mode

### LINKED ISSUE #1843: Document: feat(cmd_all): support `single_node` mode
This issue tracks the documentation update needed for the merged PR #14951.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/14951
Source PR Merged At: 2024-02-05T09:52:38Z

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

  preview: https://pr-1882.d2fbku9n2b6wde.amplifyapp.com/docs/dev/get-started/

- **Notes**

  - Let me know if any further changes are needed, thanks!

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/14951

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1843

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by kwannoel — APPROVED** [review]
LGTM, thanks!

File: docs/get-started.md [hunk 1]
```diff
@@ -14,17 +14,25 @@ This guide is designed to help you get up and running with RisingWave quickly an
 ## Start RisingWave
 
 :::info
-The following options start RisingWave in playground mode, which is designed for quick tests. In this mode, data is stored in memory and the service automatically terminates after 30 minutes of inactivity, deleting all user data. Due to limited memory capacity, resource-intensive operations may lead to out-of-memory (OOM) errors.
+The following options start RisingWave in the single-node mode. In this mode, data is stored in the file system and the metadata is stored in the embedded SQLite database.
```
Comment (hengm3467) [inline docs/get-started.md]: @kwannoel Do we want to refer to it as "standalone mode" instead?

Comment (kwannoel) [reply docs/get-started.md]: Yes, let's refer to it as standalone mode.

**Review by ShanlanLi — APPROVED** [review]
LGTM

**Pull request merged**